pkg/chartmuseum/server/multitenant/server_test.go

```diff
@@ -641,6 +659,14 @@ func (suite *MultiTenantServerTestSuite) TestBadChartUpload() {
 	suite.Equal(400, res.Status(), "400 POST /api/charts")
 }
 
+func (suite *MultiTenantServerTestSuite) TestSemver2Validation() {
+	content, err := ioutil.ReadFile(badTestSemver2Path)
+	suite.Nil(err, "no error opening test path")
+	body := bytes.NewBuffer(content)
+	res := suite.doRequest("semver2", "POST", "/api/charts", body, "")
+	suite.Equal(400, res.Status(), "403 POST /api/charts bad semver validation")
```

Comment:
"400 POST" you mean?

Reply:
sorry , I still test my code to pass the testing locally, but it always fails at some point ,  so , I push some bad code to upstream to check whether it will trigger the pr stage CI  XD . I will fixed up it back , but could I ask how can I trigger the pr stage CI  correctly ?